WordPress/src/main/res/layout/qrcodeauth_error_view.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/error_container"
```

Comment:
I'm aware that UI is not ready for the flow. We might want to reuse `ActionableEmptyView` for the error layout if we can eliminate the cancel button and use a cross/ back button in a toolbar. Let's just keep it in the back of our minds for the future.

Reply:
@ashiagr - Funny that you say that, I had every intention of using ActionableEmptyView, but I need the "cancel" link and not the back button to match the iOS design. I "borrowed" AEV and adjusted it for this use case. 👍